# Weather Message
std_msgs/Header header

# Wind speed in m/s
float32 wind_speed

# Wind direction in degrees
float32 wind_direction
